File: pdf_focus_net/PDF Focus .Net Trial (6.9.7.6)/Code samples/VB.Net/PDF to Text/Convert PDF file to Text file/Sample.vb
Imports System.IO
Imports System.Drawing.Imaging
Imports System.Collections.Generic
Imports SautinSoft

Module Sample

    Sub Main()
        Dim pdfFile As String = "..\..\..\..\Potato Beetle.pdf"
        Dim textFile As String = Path.ChangeExtension(pdfFile, ".txt")

        'Convert PDF file to Text file
        Dim f As New SautinSoft.PdfFocus()
        'this property is necessary only for registered version
        'f.Serial = "XXXXXXXXXXX"

        f.OpenPdf(pdfFile)

        If f.PageCount > 0 Then
            Dim result As Integer = f.ToText(textFile)

            'Show Text document
            If result = 0 Then
                System.Diagnostics.Process.Start(textFile)
            End If
        End If
    End Sub
End Module

File: pdf_focus_net/PDF Focus .Net Trial (6.9.7.6)/Code samples/CSharp/PDF to Text/Convert PDF file to Text file/Sample.cs
using System;
using System.IO;

namespace Sample
{
    class Sample
    {
        static void Main(string[] args)
        {
            string pdfFile = @"..\..\..\..\..\Potato Beetle.pdf";
            string textFile = Path.ChangeExtension(pdfFile, ".txt");

            //Convert PDF file to Text file
            SautinSoft.PdfFocus f = new SautinSoft.PdfFocus();
            //this property is necessary only for registered version
            //f.Serial = "XXXXXXXXXXX";

            f.OpenPdf(pdfFile);

            if (f.PageCount > 0)
            {
                int result = f.ToText(textFile);

                //Show Text document
                if (result == 0)
                {
                    System.Diagnostics.Process.Start(textFile);
                }
            }
        }
    }
}
